packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -100,6 +100,33 @@ export default ( blockData, baseTree, userTree ) => {
 		);
 	};
 
+	const flattenTree = ( input, prefix, token ) => {
+		let result = [];
+		Object.keys( input ).forEach( ( key ) => {
```

Comment:
Minor non-blocking suggestion: I guess using reduce instead of Object.keys + forEach would simplify a little bit the code here.